### DISCUSSION TITLE
How to make a preload?

### DISCUSSION THREAD

**Discussion by Suyens** [cables-gl/cables_docs#269] [Answers & Questions] [answered]
Hi, I made different scenes and use RouteTrigger to switch between them, but when I switch from scene 1 to scene 2, I get stuck for a while. How can I deal with this?

**Comment by steam0r** [comment, marked as answer]
have you taken a look at

https://dev.cables.gl/op/Ops.Patch.LoadingStatus_v2

it loads all you assets that are set to active and then outputs a trigger and a boolean that you
can use to show https://dev.cables.gl/op/Ops.Html.LoadingIndicator while loading, for example.

if this is more about rendering and not loading assets maybe take a look at https://cables.gl/op/Ops.TimeLine.DemoPrerender

make sure to read the documentation. it tries to load all you assets and basically pre-render them.
